local-volume/provisioner/cmd/main.go

```diff
@@ -43,6 +44,7 @@ var (
 )
 
 func main() {
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
This may be causing mac osx build to fail?

Reply:
helm install script, fixed in https://github.com/kubernetes-incubator/external-storage/pull/806.